src/components/ClusterOverview/TopConsumers/tests/TopConsumers.test.js

```diff
@@ -1,14 +1,98 @@
 import React from 'react';
-import { render } from 'enzyme';
+import { mount, render } from 'enzyme';
 
+import { selectDropdownItem } from '../../../../tests/enzyme';
 import { TopConsumers } from '../TopConsumers';
 import { consumersData } from '../fixtures/TopConsumers.fixture';
+import { NODES, BY_MEMORY, BY_STORAGE, BY_NETWORK } from '../strings';
 
 const testTopConsumersOverview = () => <TopConsumers {...consumersData} />;
 
+const selectType = (component, newType) => {
+  const typeDropdown = component.find('#type-dropdown');
+  selectDropdownItem(typeDropdown, newType);
+};
+
+const selectSortBy = (component, sortBy) => {
+  const sortByDropdown = component.find('#sort-by-dropdown');
+  selectDropdownItem(sortByDropdown, sortBy);
+};
+
 describe('<TopConsumers />', () => {
-  it('renders correctly', () => {
+  it('renders correctly (PODS_AND_VMS BY_CPU)', () => {
     const component = render(testTopConsumersOverview());
     expect(component).toMatchSnapshot();
   });
+
+  it('renders correctly (PODS_AND_VMS BY_MEMORY)', () => {
+    const component = mount(testTopConsumersOverview());
```

Comment:
mounting this card results in a big snapshots which are hard to follow for future updates. I'd suggest to remove the test updates from this PR and use https://github.com/kubevirt/web-ui-components/pull/432/files#diff-f866a1899292fa89cfdd5c4b02bfb306R39